src/tetris.py

```diff
@@ -26,8 +26,8 @@ class GameState(Enum):
 
 class Tetris:
     def __init__(self) -> None:
-        self.grid = TetrisGrid(10, 10)
-        self.virtual_grid = TetrisVirtualGrid(10, 10, self.grid.sync)
+        self.grid = TetrisGrid(20, 10)
+        self.virtual_grid = TetrisVirtualGrid(20, 10, self.grid.sync)
         self.shape_generator = RandomShapeGenerator(0, 4)
         self.gravity = Gravity(self.move_down)
```

Comment:
buradaki degerleri bu sinifin icinde olusturmaktansa parametre olarak alip oyunu baslatirken verebiliriz

Reply:
adamın biri gelip 3 x 3 yapar L ve I şekli geldiğinde oyun direk biter . onun kontrol mekanizmasını yapıp bu sorunu aşmak yerine statik alanda oynanmasını tercih ediyorum .

Reply:
yaptım birşey ama senin istediğin gibi mi oldu tam anlam veremedim ne istediğine :)

Reply:
kod ornegi paylastim